lib/apiservers/engine/backends/container.go

```diff
@@ -565,11 +565,10 @@ func (c *Container) ExecExists(eid string) (bool, error) {
 	return true, nil
 }
 
-// docker's container.stateBackend
-
 // ContainerCreate creates a container.
 func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (containertypes.ContainerCreateCreatedBody, error) {
 	defer trace.End(trace.Begin(""))
+	// TODO: Add Op logging to this operation. This is the start of container create operation and op logging here is very valuable.
```

Comment:
This is a general statement for all of the direct rest endpoint if only from an audit logging perspective.
Either add the logging or drop the comment.

Reply:
I will not be adding the logging, I will remove the comment. Adding op logging is defintely badly needed, but will require a decently sized refactor.